include/libtorrent/ip_filter.hpp

```diff
@@ -41,21 +41,12 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <vector>
 #include <cstdint>
 #include <tuple>
-#include <iterator> // for next
 #include <limits>
 
 #include "libtorrent/address.hpp"
-#include "libtorrent/assert.hpp"
 
 namespace libtorrent {
 
-// hidden
-inline bool operator<=(address const& lhs
```

Comment:
hi @arvidn, how one can be sure this is not used (or is it?)

Reply:
I imagine if it builds it's not used.